components/mdx-components.tsx

```diff
@@ -48,6 +49,9 @@ export const components: Record<string, FC<Record<string, any>>> = {
   GetstartedCard() {
     return <GetstartedCard />
   },
+  GetstartedCard2() {
```

Comment:
We won't be needing GetStartedCard2. If there is a difference between this and the GetstartedCard, please conditionally render a version of the card or use [the early return ](https://javascript.plainenglish.io/early-return-with-react-hooks-f96fa4a33124)to return another variant of the card.